domain-get-subdomains/function.json

```diff
@@ -1,12 +1,12 @@
 {
   "bindings": [
     {
-      "route": "domains/subdomains/{id}",
+      "route": "v1/domains/subdomains/{id}",
```

Comment:
I'm assuming this means we're adopting a api endpoint version schema?
ie; `v1/domains/subdomains/0x123` -> `v2/domains/subdomains/0x123`

How long will we be supporting old versions, such as legacy?

Reply:
I imagine we'd remove all the legacy endpoints as soon as the znsSDK was updated to consume this v1 getSubdomains